solutions/bio1/Juggluging/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { Link, SCodeBlock, SText, STitle } from "components";
+
+export const Juggluging = () => {
+    return (
+        <>
+            <STitle>Observation</STitle>
+            <SText>
+                Notice that the problem is asking for the shortest
+                number of steps required to reach one of a set of
+                states from a given state. This is a classic
+                shortest path problem, which can be solved using
+                Breadth First Search (BFS).
```

Comment:
Check this out:
https://github.com/myst-6/ukoly/blob/a06a034d5014ad25c9c60b9611d541f566673bf1/solutions/bio1/PrimeConnections/index.tsx#L11
https://github.com/myst-6/ukoly/blob/a06a034d5014ad25c9c60b9611d541f566673bf1/solutions/bio1/DreamingSpires/index.tsx#L8

In these other two places on the site, we've written `breadth-first search`, instead of `Breadth First Search`. So maybe we should write this instead.

Reply:
OK sure. However, you might want to review PrimeConnections because that statement "Since the graph is undirected, we can use a breadth-first search" is just plain wrong????